Automattic/wp-calypso
# Add E2E test for Premium themes in with-theme onboarding flow

This adds a new e2e test that covers user creation, onboarding, theme installation and account deletion for the with-theme flow with a premium theme.

Currently blocked by the changes introduced in https://github.com/Automattic/wp-calypso/pull/83863. 



Related to https://github.com/Automattic/dotcom-forge/issues/4148

## Proposed Changes

* Adds a new e2e test for a premium theme using the `with-theme` flow

## Testing Instructions



* N/A - Check the PR's e2e tests or run the test locally following the instructions in the [docs](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/README.md#quick-start).
* To execute the test for a given URL, execute it with the `CALYPSO_BASE_URL=https://calypso live link from the PR`: e.g. 
``CALYPSO_BASE_URL=https://container-tender-burnell.calypso.live/ yarn workspace wp-e2e-tests test -- test/e2e/specs/onboarding/signup__with-theme-premium.ts
``

## Pre-merge Checklist
